server/events.go

```diff
@@ -833,46 +833,87 @@ func (s *Server) leafNodeConnected(sub *subscription, _ *client, subject, reply
 }
 
 // Common filter options for system requests STATSZ VARZ SUBSZ CONNZ ROUTEZ GATEWAYZ LEAFZ
-type ZFilterOptions struct {
-	Name    string `json:"name"`
-	Cluster string `json:"cluster"`
-	Host    string `json:"host"`
+// These options are
+type EventFilterOptions struct {
+	Name    string `json:"server-name,omitempty"` // filter by server name
```

Comment:
@ripienaar , got a better naming suggestion? just server?

Reply:
json keys tend to be snake case `server_name`, otherwise seems ok

Reply:
fixed